src/pages/blog.tsx

```diff
@@ -58,14 +58,18 @@ const BlogPage = ({ data, location }: BlogPageProps) => {
   const { t } = useTranslation();
   const blogPosts = data.allMarkdownRemark.nodes;
   const { language, changeLanguage } = useI18next();
+  const breadcrumb = [
+    { pathname: '/', label: 'Satellytes' },
+    { pathname: '/blog', label: 'Blog' },
```

Comment:
To be completely future-proof we could make use of the translations here:
```suggestion
    { pathname: '/blog', label: t('navigation.blog') },
```

I do realize that the `SEO` element on line 73 receives this same string hardcoded but I consider it a best practice to make everything translatable (except maybe "Satellytes"). Maybe we will add a Mongolian version in the future and they have a special word for "Blog"?